### PR TITLE
Add basic API endpoints, functionality not implemented

### PR DESCRIPTION
**Pull request by latenssi**
- Add project boilerplate (docker-compose setup, makefile, configs etc.)
- Add initial API endpoints

### CODE REVIEW

**Comment by aphelionz** [comment]
@nanuuki might you be able to give this a once over?

File: service/http/handlers.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+// Create a distribution
+func HandleCreateDistribution(logger *log.Logger, app *app.App) http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		// Check body is not empty
+		if err := checkNonEmptyBody(r); err != nil {
+			handleError(rw, logger, err)
+			return
+		}
+
+		var dist CreatDistributionRequest
```
Comment (nanuuki) [inline service/http/handlers.go:22]: typo
```suggestion
		var dist CreateDistributionRequest
```

File: service/http/types.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	"github.com/flow-hydraulics/flow-pds/service/app"
+	"github.com/onflow/flow-go-sdk"
+)
+
+type CreateDistributionResponse struct {
+	// TODO (latenssi)
+	DistributionId string `json:"distributionId"`
+}
+
+type CreatDistributionRequest struct {
```
Comment (nanuuki) [inline service/http/types.go:13]: typo
```suggestion
type CreateDistributionRequest struct {
```

File: service/http/types.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+func DistributionFromApp(appDist app.Distribution) Distribution {
+	// TODO (latenssi)
+	return Distribution{
+		Issuer: appDist.Issuer.Hex(),
+	}
+}
+
+func (d *CreatDistributionRequest) ToApp() app.Distribution {
```
Comment (nanuuki) [inline service/http/types.go:30]: typo
```suggestion
func (d *CreateDistributionRequest) ToApp() app.Distribution {
```

**Review by nanuuki — CHANGES_REQUESTED** [review]

**Review by nanuuki — APPROVED** [review]

**Pull request merged**